packages/central-server/src/apiV2/tasks/CreateTask.js

```diff
@@ -21,6 +21,10 @@ export class CreateTask extends CreateHandler {
   }
 
   async createRecord() {
-    await this.insertRecord();
+    const { models } = this;
+    const task = await models.task.create(this.newRecordData);
+    return {
+      id: task.id,
```

Comment:
This is so we can get the id from the result in `datatrak-web-server` to create comments